libs/gpg45-evaluator/src/main/java/uk/gov/di/ipv/core/library/gpg45/Gpg45ProfileEvaluator.java

```diff
@@ -45,9 +49,9 @@ public Optional<Gpg45Profile> getFirstMatchingProfile(
     }
 
     public Gpg45Scores buildScore(List<VerifiableCredential> vcs) {
-
+        var deduplicatedVcs = deduplicateVcsByDocumentType(vcs);
         var identityChecks =
-                vcs.stream()
+                deduplicatedVcs.stream()
```

Comment:
This has slightly different semantics, as it'll exclude _all_ scores from common document types, not just the 'evidence' scores.

The DCMAW VC also includes a verification score, which we don't want to lose, and there's also some chance we'll be splitting the DCMAW VC into two, so we'd have:
- one DL VC with strength + validity scores
- one DL VC with a verification score
- (perhaps some other DL VC with a strength + validity score)

In this scenario we only want to allow one 'evidence' item (i.e. strength and validity) but still accept the verification score.

You might have to keep the full `identityCheckCredential` here and then the doc type will be available when building the evidences (and you can push the deduplication there).

Reply:
Ah ok thanks,  will take a look at this now